docs/en/02_Developer_Guides/10_Email/index.md

```diff
@@ -14,7 +14,7 @@ covers how to create an `Email` instance, customise it with a HTML template, the
 SilverStripe provides an API over the top of the [SwiftMailer](http://swiftmailer.org/) PHP library which comes with an
 extensive list of "transports" for sending mail via different services. 
 
-Out of the box, SilverStripe will use the built-in PHP `mail()` command via the `Swift_MailTransport` class. If you'd
+Out of the box, SilverStripe will use the built-in PHP `mail()` command via the `Swift_SendmailTransport` class. If you'd
```

Comment:
This isn't what the sendmail transport does, it interacts with the sendmail binary on the operating system - https://swiftmailer.symfony.com/docs/sending.html#transport-types

I think we will either need to pull in transport that uses `mail()` or write our own :/

Reply:
see https://github.com/swiftmailer/swiftmailer/issues/866

Reply:
Yeah this isn't so great

We cannot guarantee if a server has the `sendmail` binary is available nor if it's enabled in php.ini `sendmail_path` e.g. some webhosts will use [postfix](http://www.postfix.org/start.html) as the backend for mail()

It means that when a site upgrades to say framework 4.9 (assuming this change is merged in) then a site that sends email may suddenly (and probably silently) stop sending email

> I think we will either need to pull in transport that uses mail() or write our own :/

Yeah it seems like this might need to happen

Pretty much, I don't see any way around having add back support for the `mail()` function